daemon/CMakeLists.txt

```diff
@@ -52,6 +62,23 @@ target_link_libraries(libbpftime_daemon PRIVATE
     PUBLIC
     rt
 )
+else()
+target_include_directories(libbpftime_daemon PRIVATE
+    ${CMAKE_CURRENT_BINARY_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}/../vm/include
+    ${CMAKE_CURRENT_SOURCE_DIR}/../runtime/include
+)
+target_link_libraries(libbpftime_daemon PRIVATE
+    elf
+    z
+    spdlog::spdlog
+    runtime
+    PUBLIC
+    rt
+)
+endif()
```

Comment:
Same as above... or would an empty `LIBBPF_INCLUDE_DIR` just work without branching for including directories?

Reply:
have removed flag from this `CMakeLists.txt` file.